rubocop.yml

```diff
@@ -542,6 +542,9 @@ Layout/DefEndAlignment:
   - start_of_line
   - def
 
+Layout/BeginEndAlignment:
+  EnforcedStyleAlignWith: start_of_line
```

Comment:
This is the default config, so we can just enable the cop.

```suggestion
  Enabled: true
```

Reply:
Ok.  I wasn't sure if we wanted to rely on the default not changing.